apps/server/src/modules/h5p-editor/h5p-editor-test.module.ts

```diff
@@ -0,0 +1,37 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { ALL_ENTITIES } from '@shared/domain';
```

Comment:
Can you please replace on all modules you have in this PR ALL_ENTITIES to the minimal set of entities do you need? I think, user, account, school, systemId or so.

Reply:
Using Account, Role, School, SchoolYear, User now